controllers/orderCtrller.js

```diff
@@ -187,11 +190,49 @@ module.exports = {
       // 清除購物車 items
       await CartItem.destroy({ where: { CartId: cart.id } })
 
-      res.redirect('/products')
+      res.redirect('/orders/success')
+
+    } catch (err) {
+      console.error(err)
+      res.status(500).json({ status: 'serverError', message: err.toString() })
+    }
+  },
+
+  async getSuccessOrder(req, res) {
+    try {
+      const user = await User.findByPk(req.user.id, {
+        include: [
+          { model: Order, include: [
+            { model: Product, as: 'products', include: 'Images'}, Delivery]
+        }],
+        order: [['Orders', 'id', 'DESC']]
+      })
+
+      const order = user.Orders[0]
+      let subtotal = 0
+      const orderProducts = order.products
+      orderProducts.forEach(product => {
+        product.mainImg = product.Images.find(img => img.isMain).url
+        product.priceFormat = product.OrderItem.price.toLocaleString()
+        product.subPriceFormat = (product.OrderItem.price * product.OrderItem.quantity).toLocaleString()
+        subtotal += (product.OrderItem.price * product.OrderItem.quantity)
+      })
+
+      const subtotalFormat = subtotal.toLocaleString()
+      const shippingFee = order.Delivery.price
+      const receiver = order.receiver.split(",")
+      const address = order.address.split(",")
+
+      // 下訂時間
+      const orderTime = moment(order.createdAt).format('YYYY/MM/DD HH:MM')
```

Comment:
order.createdAt = 2020-01-09T06:47:42.000Z
```suggestion
      const orderTime = moment(order.createdAt).format('YYYY/MM/DD h:mm')   //2020/01/09 2:47
```
```suggestion
      const orderTime = moment(order.createdAt).format('LLL') //2020年1月9日 14:47
```